src/PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs

```diff
@@ -625,7 +640,9 @@ public static string GetTargetFrameworkString(EnvDTEProject envDTEProject)
                 ||
                 string.Equals(envDTEProject.Kind, NuGetVSConstants.FsharpProjectTypeGuid, StringComparison.OrdinalIgnoreCase)
                 ||
-                string.Equals(envDTEProject.Kind, NuGetVSConstants.JsProjectTypeGuid, StringComparison.OrdinalIgnoreCase))
+                string.Equals(envDTEProject.Kind, NuGetVSConstants.JsProjectTypeGuid, StringComparison.OrdinalIgnoreCase)
+                ||
+                string.Equals(envDTEProject.Kind, NuGetVSConstants.CpsProjectTypeGuid, StringComparison.OrdinalIgnoreCase))
```

Comment:
It looks like the right fix might be in CPS. If you've already created the packages.config file on disk then IsDocumentInProject should have found it. I wonder if the fact that you're passing in an empty array as the 3rd argument is throwing CPS off. I'll look into it.